client/jetpack-connect/remote-credentials.js

```diff
@@ -133,15 +134,13 @@ export class OrgCredentialsForm extends Component {
 	getHeaderText() {
 		const { translate } = this.props;
 
-		return translate( 'Add your website credentials' );
+		return translate( 'Add your self-hosted WordPress credentials' );
 	}
 
 	getSubHeaderText() {
 		const { translate } = this.props;
 		const subheader = translate(
-			'Add your WordPress administrator credentials ' +
-				'for this site. Your credentials will not be stored and are used for the purpose ' +
-				'of installing Jetpack securely. You can also skip this step entirely and install Jetpack manually.'
+			'Your log in credentials are used for the purpose of securely auto-installing Jetpack and will not be stored.'
```

Comment:
Adding @michelleweber's suggestion here:

```suggestion
			'Your login credentials are used for the purpose of securely auto-installing Jetpack and will not be stored.'
```